### PR TITLE
global: release checklist

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by slint** [comment]
Apparently [this doctest](https://github.com/inveniosoftware/invenio-oaiserver/blob/master/invenio_oaiserver/__init__.py#L108) is running out of the application context...

Also this [line is too long](https://github.com/inveniosoftware/invenio-oaiserver/blob/master/invenio_oaiserver/__init__.py#L150), you can split it with `\` safely

File: README.rst [hunk 1]
```diff
@@ -45,5 +45,4 @@ Invenio module that implements OAI-PMH server.
 
 *This is an experimental developer preview release.*
```
Comment (lnielsen) [inline README.rst]: You can remove this line as well.

**Comment by krzysztof** [comment]
--doctest-modules was removed from pytest.ini options:
addopts = --pep8 --doctest-glob="*.rst" ~~--doctest-modules~~ --cov=invenio_oaiserver --cov-report=term-missing

and created as a separate issue: https://github.com/inveniosoftware/invenio-oaiserver/issues/133

**Comment by krzysztof** [comment]
@slint removed the debug code

**Pull request merged**